### DISCUSSION TITLE
So far I used the eventTitleSize parmeter to enlarge the event name.

### DISCUSSION THREAD

**Discussion by Korte68** [totaldebug/atomic-calendar-revive#1238] [Support] [answered]
How can I enlarge the event title?
Can I remove the day number before the event? I would like to display only the event name.

Thank you

**Comment by marksie1988** [comment, marked as answer]
Below should do what you want, just change the `font-size` to what you need

```
card_mod:
  style: |
      .event-title{
        font-size: 20px;
      }
      .single-event-container {
        display: block !important;
      }
      .event-left {
        visibility: hidden !important;
      }
```

**Reply by Korte68** [reply]
Thank you, the size change working with this, but the number of the day still displayed but now in a new line above the events.

**Reply by marksie1988** [reply]
Oh sorry you need a . In front of event-left. I'll update the example above.

**Reply by Korte68** [reply]
Perfect! Thank you very much.

**Reply by Korte68** [reply]
May I have an another question? After the page is loaded (where this card is exist) it takes more than 5s to display the events. Why does it take so long? Is there a way to cache or speed up it? Thank you

**Reply by marksie1988** [reply]
It has to connect to the home assistant API for each of the calendars, then it sorts all the events etc.

**Reply by Korte68** [reply]
I see, but maybe a cache would help...then after 5sec update the cache and the display?

**Reply by Korte68** [reply]
hi, sorry, but it was working for a while, but now it is not working again...something changed?

**Reply by marksie1988** [reply]
Nothing has changed with the styling.

**Reply by marksie1988** [reply]
Just tested and it's still working for me

**Reply by Korte68** [reply]
you are right. it was my mistake.
on more question...How can I align the text to the center?
 Thank you.
